### DISCUSSION TITLE
I want to download a full subreddit, with its posts, comments (100%), images and videos.

### DISCUSSION THREAD

**Discussion by StableInfo** [Serene-Arc/bulk-downloader-for-reddit#890] [Q&A] [answered]
I can use this python3 -m bdfr download path/.. --subreddit ASubreddit --submitted --authenticate --no-dupes,
But that gives you only videos, images, and posts first comments.
What is the right command to also get ALL comments of every post? Thanks

**Comment by Serene-Arc** [comment]
This requires the use of the `clone` command. The cloning command gets the posts themselves along with the thing that the posts link to (or the text if it's a text post). You'll also want `--all-comments` and  `--comment-context` to get all the comments in the posts.

**Comment by StableInfo** [comment]
Hello @Serene-Arc , isn't that argument only usable for when you want to target a user rather than a subreddit? This is the eror I get:
```
Usage: python -m bdfr download [OPTIONS] DIRECTORY

Error: No such option: --all-comments
```
```
Usage: python -m bdfr download [OPTIONS] DIRECTORY

Error: No such option: --comment-context
```

Can you test it and tell me which line of code worked for you best? I want posts AND all comments (and imagers as an option, but comments for all subreddit are most important)

**Reply by Serene-Arc** [reply, marked as answer]
I said you need to use the clone command. Read through the README. But yes, my mistake. The `--all-comments` is not correct, it's just the `--comment-context` option.

**Reply by StableInfo** [reply]
Great, it is downloading.
I think I am limited by the 1000 limit for APIs right? Is there a way to "restart" the download with another account and get 1000 extra downloads? By avoiding duplicates?  I would like to download 2 months of content, do you think its possible?

**Reply by Serene-Arc** [reply]
There's no way to get past the 1000 post limit, that's reddit's thing, not ours. We can't change it or bypass it. The exception is if you're after your own data, in which case a GPDR data request will get more than 1000 posts.
